tests/conftest.py

```diff
@@ -1206,12 +1210,24 @@ def flow_cells_demux_all_dir(demultiplex_fixtures: Path) -> Path:
     return Path(demultiplex_fixtures, "flow_cells_demux_all")
 
 
-@pytest.fixture(name="demux_results_not_finished_dir")
+@pytest.fixture
 def demux_results_not_finished_dir(demultiplex_fixtures: Path) -> Path:
-    """Return the path to a dir with demultiplexing results where demux has been done but nothing is cleaned."""
+    """Return the path to a dir with demultiplexing results where nothing has been cleaned."""
     return Path(demultiplex_fixtures, "demultiplexed-runs-unfinished")
 
 
+@pytest.fixture(scope="session")
```

Comment:
For these string fixtures, the scope does not matter since they are instantaneous to set up. So if you want to shave of some characters to make it more readable, this works just as well
```python
@pytest.fixture
def hiseq_x_flow_cell_name() -> str:
    ...
```

Reply:
I thought we agreed on making all fixtures session-scoped by default. I think if we leave a function scope (the default) for many small fixtures like this the short times could build up

Reply:
No, it only makes a difference for path and store fixtures. String fixtures like this does not add anything. Keep it as you like 👍  If you want to play around and see what takes time 
```
pip install pytest-profiling
pip install snakeviz
pytest --profile
snakeviz prof/combined.prof
```

Reply:
Fixed for all the touched string fixtures 👍

Reply:
I am rolling back this change as many session-scoped functions depend on this fixture, which therefore can't be function-scoped